src/main/java/hudson/plugins/ec2/win/winrm/WinRMClient.java

```diff
@@ -129,7 +131,7 @@ public void sendInput(byte[] input) {
 
     public boolean slurpOutput(FastPipedOutputStream stdout, FastPipedOutputStream stderr) throws IOException {
         LOGGER.log(Level.FINE, () -> "--> SlurpOutput");
-        ImmutableMap<String, FastPipedOutputStream> streams = ImmutableMap.of("stdout", stdout, "stderr", stderr);
+        Map<String, FastPipedOutputStream> streams = new HashMap<>(); streams.put("stdout", stdout); streams.put("stderr", stderr);
```

Comment:
It's not an immutable but I think it's fine. We only use it internally. Maybe you can initialize to 2 elements, as they're the only elements we will use